inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_Default_HTML_general_macros.tpl

```diff
@@ -198,7 +198,7 @@
           });
           MathJax.Hub.Queue(["Typeset", MathJax.Hub]);
         });
-      })
+      }, 500)
```

Comment:
Uhm, this doesn't seem very robust...

Reply:
I see your point. I tested this code on a slower 3G network and of course, it fails. I'm looking into another solution.

Reply:
Thanks for checking! Yes, slower network is exactly the scenario I had in mind.

Perhaps there's a better hook for MathJax than DOMContentLoaded? Something in Angular's lifecycle that fires after Angular has finished its first render, or something like that...